packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/emr/emr-add-step.ts

```diff
@@ -89,6 +89,15 @@ export interface EmrAddStepProps extends sfn.TaskStateBaseProps {
    * @default - No properties
    */
   readonly properties?: { [key: string]: string };
+
+  /**
+   * The Amazon Resource Name (ARN) of the runtime role for a step on the cluster.
+   *
+   * @see https://docs.aws.amazon.com/emr/latest/APIReference/API_AddJobFlowSteps.html#API_AddJobFlowSteps_RequestSyntax
+   *
+   * @default - No step-specific role, uses EC2 instance profile role
+   */
+  readonly executionRole?: string;
```

Comment:
I'd prefer this be `executionRoleArn` but the linter didn't like it because this isn't an L1 construct.

Reply:
It doesn't like anything that ends with `arn` because of [this line](@see https://aws.amazon.com/blogs/big-data/introducing-runtime-roles-for-amazon-emr-steps-use-iam-roles-and-aws-lake-formation-for-access-control-with-amazon-emr/). I could use `eslint-disable` ?

Reply:
```suggestion
  readonly executionRoleArn?: string;
```
After maintainer feedback, let's rename this `executionRoleArn` and add the lint exception.